actfw_core/application.py

```diff
@@ -61,20 +61,34 @@ def register_task(self, task):
             raise TypeError("type(task) must be a subclass of actfw_core.task.Task.")
         self.tasks.append(task)
 
-    def run(self):
+    def start(self):
         """Start application"""
         for task in self.tasks:
             task.start()
 
+    @property
+    def is_running(self):
```

Comment:
Should we admit users to set a value via this property, or forbid and force them to use `start()/stop()`?

Reply:
forbid.

Reply:
In that case, what is the purpose of introducing this property? 

```
def is_running(self) -> bool:
    return self.running
```

is preferable.

Reply:
i forget to remove it.

Reply:
https://github.com/Idein/actfw-core/pull/21/commits/49e933f40b6afc78251a6a968d6abd45a4e1e813